src/STRPAK.CPP

```diff
@@ -406,15 +415,22 @@ char* FC strpathparts( 	// Build string from parts of a path name (for default f
 
 // Returns assembled string, equested parts which are not present in *path are omitted from result.
 {
-	char pbuf[_MAX_PATH*4];
+	char pbuf[ FILENAME_MAX*4];
```

Comment:
Did we decide that FILENAME_MAX * 4 would always be a harmless allocation? Maybe I missed the tail of that conversation.

Reply:
I forgot to bring this up in our last meeting. @nealkruis are we ok using FILENAME_MAX?

Reply:
This goes out of scope pretty quickly, so I don't think it will be a problem in terms of allocating excessive memory.